### ISSUE TITLE
[Runtime bug]: black windows open when launch sample test with runtime

### ISSUE THREAD

**Issue by Shaeiouy** [NVIDIAGameWorks/rtx-remix#415]
### Contact Details (Optional)

ofmr@msn.com

### Describe the bug

Hello,
I'm simply trying to reproduce the first RTX remix tutorial video with the "sample test". However, this doesn't work for me. opening the test sample without the runtime works, but when I place the runtime files in the test sample folder, instead of having the test sample with the runtime raytracing, I have a black screen on my side, which charges constantly. My system data indicates that my CPU (5800x) is in full activity while my GPU (3090) appears to be inactive. Therefore, for the implementation of the runtime in a game, I have absolutely nothing. I tried on Half life 2 and tomb raider 3, but I got nothing. Do you have any idea what is happening? ?

### How do you reproduce the bug?

execute the first tutorial video of RTX remix (NVIDIA Studio on Yt)
![Capture d'écran 2024-02-12 020252](https://github.com/NVIDIAGameWorks/rtx-remix/assets/159596884/dee3329a-cd63-4560-9468-e20c2562e6f9)
![Capture d'écran 2024-02-12 020341](https://github.com/NVIDIAGameWorks/rtx-remix/assets/159596884/e7c964f5-6ca6-40e7-b8e3-7da2cc029dad)
![Capture d'écran 2024-02-12 020406](https://github.com/NVIDIAGameWorks/rtx-remix/assets/159596884/49eef6eb-b230-493e-a354-c0c3228a02c0)

### What is the expected behavior?

The toolkit should, as shown in the first video of the tutorial, display the "R" of the illuminated raytracing test sample.

### Version

2024.1.1

**Comment by nvdamien** [comment]
Hi,

It looks like a runtime problem. What is your driver version please?

**Comment by Shaeiouy** [comment]
> Hi,
> 
> It looks like a runtime problem. What is your driver version please?

First of all, thank you very much for responding so quickly! I basically have the latest NVIDIA driver version: 551.23

**Comment by Shaeiouy** [comment]
Now, at 02/12/2024, the 551.52 version

**Comment by Shaeiouy** [comment]
So, what would be the solution to this runtime problem?

**Comment by NV-DM** [comment]
Does the same thing happen when you launch the Sample application directly through the executable? As in: just double click on RemixSample app instead of launching it through Remix.

**Comment by bulutthecat** [comment]
I have also been having this issue, for me it runs fine when I use the application separately without the dll's.
I also might add that the sample seems to run for two or so frames before turning back into a black window and freezing.

**Comment by Shaeiouy** [comment]
> RemixSample app



> Does the same thing happen when you launch the Sample application directly through the executable? As in: just double click on RemixSample app instead of launching it through Remix.
Yes that's it. I tried different ways to start RemixSample app, either with rtx remix, or with Windows file explorer. None work. That said, the Remixsample app works without DLL files. But there is no more raytracing there. And as a result, the application does not work when I start with a game. I tried it with Tomb Raider Anniversary (DirectX 9) and obviously, the RTXremix application did not work.

**Comment by Shaeiouy** [comment]
> Does the same thing happen when you launch the Sample application directly through the executable? As in: just double click on RemixSample app instead of launching it through Remix.

Yes that's it. I tried different ways to start RemixSample app, either with rtx remix, or with Windows file explorer. None work. That said, the Remixsample app works without DLL files. But there is no more raytracing there. And as a result, the application does not work when I start with a game. I tried it with Tomb Raider Anniversary (DirectX 9) and obviously, the RTXremix application did not work.

**Comment by Shaeiouy** [comment]
> I have also been having this issue, for me it runs fine when I use the application separately without the dll's. I also might add that the sample seems to run for two or so frames before turning back into a black window and freezing.

Yes, but when you remove the DLLs, you no longer have the raytracing effects. Indeed, for me too, when I separate the remix sample app from the DLLs, it works, but there is no RTX light in the scene. But when I add the DLLs (like the official NVIDIA tutorial on Youtube), just like you, the scene appears for a few thousandths of a second and then becomes black again, and it freezes (while my rtx 3090 is at maximum load).

**Comment by nascarfan5** [comment]
same thing is happening to me, at this point I'm considering completely resetting windows.

**Comment by nv-ajaus** [comment]
Hi @Shaeiouy @nascarfan5 @bulutthecat , sorry for the delay. Are any log files (d3d9.log, NvRemixBridge.log, RemixSample_d3d9.log) or a crash dump (.dmp file extension) generated in the directory of the game/sample app or the .trex folder? If so, can you please post them?

**Comment by Shaeiouy** [comment]
> Hi @Shaeiouy @nascarfan5 @bulutthecat , sorry for the delay. Are any log files (d3d9.log, NvRemixBridge.log, RemixSample_d3d9.log) or a crash dump (.dmp file extension) generated in the directory of the game/sample app or the .trex folder? If so, can you please post them?

Hello @nv-ajaus ,
I have a RemixSample_d3d9.log on the sample directory. Here it is:

[RemixSample_d3d9.log](https://github.com/NVIDIAGameWorks/rtx-remix/files/14895348/RemixSample_d3d9.log)

**Comment by Shaeiouy** [comment]
So?

**Comment by NV-DM** [comment]
@Shaeiouy This a bit of a guess, but I think there might be an issue in setup. For the sake of figuring it out, lets do a clean install. Could you please create a new directory and place a copy of the Sample App and Runtime like in the attached screenshot. This is before running the Sample App for the first time. Running the Sample App will create new files.
![image](https://github.com/NVIDIAGameWorks/rtx-remix/assets/130596284/98baea3a-5afa-461f-8789-099474e7606c)

**Comment by Shaeiouy** [comment]
@NV-DM Hi, as you asked, I reinstalled RTX remix and created a new directory named "SampleApp2", like yours. However, I didn't know where to create this new file. So I created it in the “remix_runtime” folder (image1). 
![image1](https://github.com/NVIDIAGameWorks/rtx-remix/assets/159596884/134dcf75-1179-40a9-b9d7-aab05eaef8dc)
Was this correct?
Then, I launched the program with "RemixSample.exe", and...the program still crashes, unfortunately (image 2). 
![image2](https://github.com/NVIDIAGameWorks/rtx-remix/assets/159596884/538848c7-751f-4b0c-9996-b2b9b587922c)
As you can see in image 3, although the program crashed, my CPU is still working intensely, while the GPU (RTX3090) is at 3%. To completely close RTXsample, I had to do it via the task manager!
![image3](https://github.com/NVIDIAGameWorks/rtx-remix/assets/159596884/317da92e-415d-4fc0-8027-8b518befee58)
 And, by returning to the “SampleApp2” folder, new files were created, as shown in image 4.
![image4](https://github.com/NVIDIAGameWorks/rtx-remix/assets/159596884/4f5adfa0-433f-437d-9351-722699fa08b0)

**Comment by NV-LL** [comment]
Hi @Shaeiouy - checking in to see if you've had success with the instructions provided in the previous comment. Let us know if you still need help! Thanks!
